legend-sdlc-generation-file-maven-plugin/src/main/java/org/finos/legend/sdlc/generation/file/FileGenerationMojo.java

```diff
@@ -51,16 +55,18 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+import static org.finos.legend.pure.generated.platform_pure_corefunctions_meta.Root_meta_pure_functions_meta_elementToPath_PackageableElement_1__String_1__String_1_;
+
 @Mojo(name = "generate-file-generations", defaultPhase = LifecyclePhase.GENERATE_SOURCES)
 public class FileGenerationMojo extends AbstractMojo
 {
-    private static final String GENERATION_SPECIFICATION_CLASSIFIER_PATH = "meta::pure::generation::metamodel::GenerationSpecification";
+    public final String FILE_PATH_DELIMETER = "/";
```

Comment:
I think you should be using the filesystem path separator instead wherever this is used.